### ISSUE TITLE
tox `packaging` target failing

### ISSUE THREAD

**Issue by msabramo** [pypa/warehouse#339]
```
❯ tox -e packaging
GLOB sdist-make: /Users/marca/dev/git-repos/warehouse/setup.py
packaging inst-nodeps: /Users/marca/dev/git-repos/warehouse/.tox/dist/warehouse-14.2.1.zip
packaging runtests: PYTHONHASHSEED='1851367937'
packaging runtests: commands[0] | check-manifest
listing source files under version control: 212 files and directories
building an sdist: warehouse-14.2.1.tar.gz: 229 files and directories
copying source files to a temporary directory
building a clean sdist: warehouse-14.2.1.tar.gz: 212 files and directories
files in version control do not match the sdist!
missing from VCS:
  docs/_build
  docs/_build/html
  docs/_build/html/_sources
  docs/_build/html/_sources/api-reference
  docs/_build/html/_sources/api-reference/index.txt
  docs/_build/html/_sources/api-reference/legacy.txt
  docs/_build/html/_sources/api-reference/xml-rpc.txt
  docs/_build/html/_sources/application.txt
  docs/_build/html/_sources/configuration.txt
  docs/_build/html/_sources/contributing.txt
  docs/_build/html/_sources/index.txt
  docs/_build/html/_sources/reference
  docs/_build/html/_sources/reference/csrf.txt
  docs/_build/html/_sources/reference/index.txt
  docs/_build/html/_sources/reference/sessions.txt
  docs/_build/html/_sources/testing.txt
  docs/_build/html/output.txt
ERROR: InvocationError: '/Users/marca/dev/git-repos/warehouse/.tox/packaging/bin/check-manifest'
___________________________________________________________________________________ summary ____________________________________________________________________________________
ERROR:   packaging: commands failed
```

**Comment by msabramo** [comment]
This should be fixed by #342.

**Comment by dstufft** [comment]
I'm going to close this, I've started porting Warehouse over to Pyramid to ease the concerns of the custom framework-ish thing I've got going on here. That work is on the pyramid branch inside this repository. This issue is no longer an issue in that new branch.
